bitnami/postgresql-ha/templates/pgpool/deployment.yaml

```diff
@@ -4,6 +4,9 @@ metadata:
   name: {{ include "postgresql-ha.pgpool" . }}
   labels: {{ include "postgresql-ha.labels" . | nindent 4 }}
     app.kubernetes.io/component: pgpool
+    {{- with .Values.pgpool.labels }}
+    {{- toYaml . | nindent 4 }}
```

Comment:
Please, could you use `"postgresql-ha.tplValue"`? and have it as a template value too.